src/core/signal.cc

```diff
@@ -31,15 +31,8 @@ module seastar;
 
 namespace seastar {
 
-static reactor& engine_or_throw() {
```

Comment:
Please, squash it with the previous commit

Reply:
sorry, done